opentracing-kafka-spring/src/main/java/io/opentracing/contrib/kafka/spring/MessageListenerMethodInterceptor.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017-2020 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.kafka.spring;
+
+import io.opentracing.*;
```

Comment:
please don't use `.*` in imports

Reply:
Alright. It would be helpful for the contributors if project owners integrated a checkstyle or similar tool into the build, if some code convention rules are important to them.